engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/types/EspTypeUtilsSpec.scala

```diff
@@ -262,7 +262,8 @@ class EspTypeUtilsSpec extends FunSuite with Matchers with OptionValues {
 
   test("should hide some ugly presented methods") {
     val classDef = singleClassDefinition[ClassWithWeirdTypesInMethods]().value
-    classDef.methods.get("methodReturningArray") shouldBe empty
+    classDef.methods.get("methodReturningArray") shouldBe
```

Comment:
probably this check should be done in another test - this one checks hidden methods

Reply:
I understand, thought just removing the test would be confusing so I modified it but I get that it doesn't really fit in here

Reply:
removed the test, you are right, there wasn't much sense for it noce arrays are included